near-rust-allocator-proxy/src/config.rs

```diff
@@ -0,0 +1,46 @@
+use crate::allocator::{
+    ENABLE_STACK_TRACE, LOGS_PATH, REPORT_USAGE_INTERVAL, SAVE_STACK_TRACES_TO_FILE,
+};
+use std::sync::atomic::Ordering;
+
+#[derive(Default)]
+pub struct AllocatorConfig {}
+
+impl AllocatorConfig {
+    /// Enable calling `backtrace` to fill out data
+    pub fn enable_stack_trace(self, value: bool) -> Self {
+        ENABLE_STACK_TRACE.store(value, Ordering::Relaxed);
```

Comment:
You don't need to store config in globas. You have access to self on the allocator: 

```
pub unsafe trait GlobalAlloc {
    unsafe fn alloc(&self, layout: Layout) -> *mut u8;
```

So config should be just fields on `ProxyAllocator`

Reply:
@matklad Ok, I'll replace it https://github.com/near/near-memory-tracker/issues/89